### DISCUSSION TITLE
FormButtonGroup 使用schema方式控制台会报warning

### DISCUSSION THREAD

**Discussion by beita-wang** [alibaba/formily#3847] [Q&A 2.x]
"FormButtonGroup": {
                                                "type": "void",
                                                "x-decorator": "FormItem",
                                                "x-component": "FormButtonGroup",
                                                "x-component-props": {
                                                    "align": "center"
                                                },
                                                "properties": {
                                                    "Submit": {
                                                        "type": "string",
                                                        "x-decorator": "FormItem",
                                                        "x-component": "Submit",
                                                        "x-component-props": {
                                                            "onSubmit": "{{(value)=>{console.log(value)}}}",
                                                            "type": "primary",
                                                            "size": "middle",
                                                            "icon": "{{icon('SaveFilled')}}"
                                                        },
                                                        "x-decorator-props": {}
                                                    }
                                                }
                                            }
**warning**: The tag <FormButtonGroup> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.

**Comment by Landon-CN** [comment]
你创建SchemaField的代码发上来呗，估计是没注册

**Reply by beita-wang** [reply]
我的意思是组件已经创建了 界面也可以看到，但是用schema的方式写FormButtonGroup会有warning

**Reply by Landon-CN** [reply]
你整个demo吧，不然都不知道哪里出问题了

**Reply by beita-wang** [reply]
上图就是demo，json schema按上面的方法写，渲染就会有warning

**Reply by Landon-CN** [reply]
https://codesandbox.io/s/nervous-wildflower-7dmphf?file=/App.tsx 无法复现
